CM7/Core/Inc/queues.h

```diff
@@ -0,0 +1,15 @@
+#ifndef QUEUES_H
+#define QUEUES_H
+
+#include "cmsis_os.h"
+#include "gatedriver.h"
+
+#define PHASE_CURRENT_QUEUE_SIZE    8
+
+typedef struct {
+    uint32_t current[GATEDRV_NUM_PHASES];
+} phase_current_t;
+
+extern osMessageQueueId_t phase_current_queue;
```

Comment:
We can leave this in the PR rn, but I think eventually it should just reference the queue set up in the foc control block, but we gotta wait for that PR to be merged in. Just a heads up that this works for now but might change